.github/workflows/CI.yml

```diff
@@ -65,12 +77,12 @@ jobs:
       run: |
         xvfb-run --auto-servernum python atest/run.py --zip firefox
 
-    - name: Run tests with Selenium Grid
-      if: matrix.python-version == '3.8' && matrix.rf-version == '3.2.2' && matrix.python-version != 'pypy-3.7'
-      run: |
-        wget --no-verbose --output-document=./selenium-server-standalone.jar http://selenium-release.storage.googleapis.com/3.141/selenium-server-standalone-3.141.59.jar
-        sudo chmod u+x ./selenium-server-standalone.jar
-        xvfb-run --auto-servernum python atest/run.py --zip headlesschrome  --grid True
+    # - name: Run tests with Selenium Grid
```

Comment:
Should all this be commented out? These will not run anyway because 'matrix.rf-version 3.2.2' is not in the list. Maybe also add a comment like the others?